marketplace/connect/tasks.py

```diff
@@ -0,0 +1,27 @@
+from .client import ConnectProjectClient, WPPRouterChannelClient
+from marketplace.celery import app as celery_app
+
+
+@celery_app.task(name="create_channel")
+def create_channel(user: str, project_uuid: str, data: dict, channeltype_code: str):
```

Comment:
Why does this task exist? could this client not be called directly in the view?
The same question applies to the two tasks below.